test/index.test.js

```diff
@@ -0,0 +1,44 @@
+const { searchRedditPostsByTopicPromise } = require('../build/index');
```

Comment:
Our imports should be organized into the following sections:
1. standard node libraries
2. external npm libraries
3. our own imports

Each section should be alphabetized.

### Why?
Standardizing organization makes it easier for developers to read and understand a project.

In addition, when collaborating with other developers on large projects, merge conflicts can pop up in the import section if there's not a pattern to follow.